msteams-platform/concepts/build-and-test/develop-your-apps-with-teams-toolkit.md

```diff
@@ -28,6 +28,9 @@ Following are the steps to open your app in Visual Studio Code:
 
     A dialog is displayed.
 
+    > [!NOTE]
+    > Ensure that you install the latest version of [Teams Toolkit for Visual Studio Code](../../toolkit/install-Teams-Toolkit.md) to allow the app to open.
```

Comment:
To enable your app to open, install the latest recent version of [Teams Toolkit for Visual Studio Code](../../toolkit/install-Teams-Toolkit.md).